config/DBPediaSearch.properties

```diff
@@ -36,7 +36,7 @@ kb.insert.root.class=http://www.w3.org/2002/07/owl#Thing
 kb.insert.label=http://www.w3.org/2000/01/rdf-schema#label
 kb.insert.alternative.label=http://www.w3.org/2000/01/rdf-schema#label
 kb.insert.subclass.of=http://www.w3.org/2000/01/rdf-schema#subClassOf
-kb.insert.instance.of=http://www.w3.org/1999/02/22-rdf-syntax-ns#type
+kb.insert.instance.of=https://www.w3.org/1999/02/22-rdf-syntax-ns#type
```

Comment:
I was testing this change for #296. I will revert it, because it is more correct with http.